Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Create.cshtml

```diff
@@ -327,8 +327,8 @@
                         <p tabindex="0" id="model-desc">The record you are creating for <strong>@(warning.MessageParameters[0])</strong> in <strong>@(warning.MessageParameters[1])</strong> may duplicate existing record(s): <strong>@(warning.MessageParameters[2])</strong>
                     </div>
                     <div class="button-row">
-                        <button type="submit" name="@nameof(Model.ActionStep)" value="@Model.ActionStep" class="govuk-button">Proceed</button>
-                        <button type="submit" name="@nameof(Model.ActionStep)" value="@Model.PreviousStep" class="govuk-button govuk-button--secondary" >Cancel</button>
+                        <button type="submit" id="proceed-button" name="@nameof(Model.ActionStep)" value="@Model.ActionStep" class="govuk-button">Proceed</button>
+                        <button type="submit" id="cancel-button" name="@nameof(Model.ActionStep)" value="@Model.PreviousStep" class="govuk-button govuk-button--secondary" >Cancel</button>
```

Comment:
I note that there are multiple elements with the same `id` -- if you're happy that they'll never be on the same page together, I'm happy :)

Reply:
Yes. proceed, cancel, submit these buttons are seen in every page but not seen on the same page together